tinker/tinker_controller.py

```diff
@@ -130,7 +130,7 @@ def init_user():
                 day_is_passed = True
 
             # if not production, then clear our session variables on each call
-            if dev or day_is_passed:
+            if (not session.get('admin_viewer', False)) and (dev or day_is_passed):
```

Comment:
no, that logic was already there. I just added the part to check if we were viewing with admin_viewer == True